news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Community Mentorship Program Winter 2017 Signups Now Open"
+date: 2017-10-08 12:00:00 +0000
+---
+Have you ever opened up the editor and just looked at everything it offers in confusion? Do you have no idea where to even begin beatmapping? Worry no further, the Community Mentorship Program might be the place for you!
+
+![](/wiki/shared/news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open/banner.png)
```

Comment:
Add alt text on L8

Reply:
legitimately no newspost header does this - i think there's a reason for that